backend/src/JWT/Model/JWTToken.php

```diff
@@ -20,33 +20,16 @@
 
 class JWTToken
 {
-    private ?string $iss;
-    private ?string $aud;
-    private ?string $sub;
-    private ?string $room;
-    private ?FeaturesPayload $features;
-    private ?\DateTimeImmutable $nbf;
-    private ?UserPayload $user;
-    private ?HeaderPayload  $headerPayload;
-
     public function __construct(
-        string $iss,
-        string $aud,
-        string $sub,
-        string $room,
-        UserPayload $user,
-        ?\DateTimeImmutable $nbf = null,
-        ?HeaderPayload $headerPayload = null,
-        ?FeaturesPayload $featuresPayload = null
+        private readonly ?string $iss,
```

Comment:
this looks strange too, is it string or ?string?

Reply:
You're right :) thank you for the whole review!!